get-forms/index.html

```diff
@@ -3,11 +3,41 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Get form</title>
     <meta name="description" content="My Github Search form" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-    <form action="https://github.com/search" method="get"></form>
+    <div class="form">
+      <form action="https://github.com/search" method="get">
+        <div>
+          <label for="q">Search Field</label>
+          <input type="text" name="q" />
```

Comment:
I suggest inserting "required" in this input, preventing the user does a blank search.

Reply:
Thank you, Pedro done ✔️